dark/reads.py

```diff
@@ -133,6 +135,26 @@ def walkHSP(self, hsp):
             readOffset += 1
             subjectOffset += 1
 
+    def checkAlphabeth(self, count=10):
+        """
+        A function which checks whether the sequence in a L{dark.Read} object
+        corresponds to its readClass. For AA reads, more testing is done in
+        dark.Read.AARead.checkAlphabeth.
+
+        @param count: A C{int} of how many amino acids should be considered.
+        @return: C{True} if the sequence corresponds from its readClass or
```

Comment:
This needs to be updated.  C{True} if the alphabet characters in the first C{count} positions of sequence is a subset of the allowed alphabet for this read class, or if the read class has a C{None} alphabet.